### ISSUE TITLE
Fix tag displaying

### ISSUE THREAD

**Issue by arturs105** [toggl/mobileapp#666]
Currently tag displaying in the start time entry view is broken if the tags span multiple lines.

<img width="487" alt="screen shot 2017-10-03 at 11 33 55" src="https://user-images.githubusercontent.com/15633003/31116444-cb2d68de-a82e-11e7-8a23-ba6a59c11c09.png">

**Comment by amulware** [comment]
This will be unnecessary with #685

**Comment by amulware** [comment]
Nevermind, I confused edit and start view.

**Comment by Odrakir** [comment]
I was investigating this.

So, there seems to be something the `UITextView` does with margins, which is what messes the whole layout. There might be some property we can set to zero that fixes it, but I couldn't find it.

What I did do was subclassing from `UIView`, instead of `UITextView`. Then I added the needed `attributedText` property, and finally I draw the text frame at the end of the `draw` method, as it won't do it automatically anymore.

Basically the rest of the code is more or less the same (I also removed some magic numbers)

Download this playground and see: [TextTokens](https://gist.github.com/Odrakir/36c04c70002716b03b7decdca064b3d1)

**Comment by arturs105** [comment]
I thought about this, and if we decide to derive from `UIView`, then we would need to implement text editing ourselves as well, since the tags need to appear alongside editable text.

**Comment by Odrakir** [comment]
Doh! Of course.

But now that we know where the issue is, it was easy to find a way to fix the `UITextView`. You just have to override `layoutSubviews`

```swift
    override func layoutSubviews()
    {
        super.layoutSubviews()
        textContainerInset = UIEdgeInsets.zero
        textContainer.lineFragmentPadding = 0
    }
```

This is the new playground: [here](https://gist.github.com/Odrakir/a839434f3c0182340ab311b6b20a4b24)

**Comment by Odrakir** [comment]
Finally in order to prevent the tags from being clipped, this one adds the insets to the drawing context, instead of removing them from the text view.

[Here.](https://gist.github.com/Odrakir/a6f0a392a6531d42058318cf9b0b2946)

**Comment by arturs105** [comment]
I also noticed, that tags with emojis are are being displayed weirdly.

<img width="487" alt="screen shot 2017-11-02 at 10 17 48" src="https://user-images.githubusercontent.com/15633003/32316011-175605b4-bfb7-11e7-9748-f9dff15e958a.png">
